screens/buddies.js

```diff
@@ -9,22 +9,26 @@ export default function Buddies({ navigation }) {
 
         // Basic static user data, used until backend is developed.
-        {name: 'Andrew Baker', category: 'Spiritual', goal: 'I want to attend chapel twice a week', hobby: 'Reading', email: 'coolguy@yeet.com', number: 1234573885, pic: '../assets/images/george.jpg', key: '1'},
-        {name: 'Dawson Buist', category: 'Education', goal: 'I want to read one book every week', hobby: 'Hacking', email: 'coolguy@yeet.com', number: 1234573885, pic: '../assets/images/george.jpg', key: '2'},
-        {name: 'Kelsey Yen', category: 'Spiritual', goal: 'I want to rest every day for an hour', hobby: 'Engineering', email: 'coolguy@yeet.com', number: 1234573885, pic: '../assets/images/george.jpg', key: '3'},
-        {name: 'Belina Sainju', category: 'Relaxation', goal: 'I want watch TV every day', hobby: 'Reading', email: 'coolguy@yeet.com', number: 1234573885, pic: '../assets/images/george.jpg', key: '4'},
-        {name: 'Joe Pastucha', category: 'Relaxation', goal: 'I want to watch 2 movies every week', hobby: 'Sleeping', email: 'coolguy@yeet.com', number: 2314573885, pic: '../assets/images/george.jpg', key: '5'},
-        {name: 'Nathan Strain', category: 'Spiritual', goal: 'I want to meditate daily', hobby: 'School', email: 'coolguy@yeet.com', number: 1234573885, pic: '../assets/images/george.jpg', key: '6'},
+        {name: 'Andrew Baker', category: 'Spiritual', goal: 'I want to attend chapel twice a week', hobby: 'Reading', email: 'coolguy@yeet.com', number: 1234573885, pic: 'https://scontent-ort2-2.xx.fbcdn.net/v/t1.0-9/59745462_102655627622924_4862557033871704064_o.jpg?_nc_cat=109&ccb=2&_nc_sid=09cbfe&_nc_ohc=ET9UNrS141EAX_2jGtR&_nc_ht=scontent-ort2-2.xx&oh=994cc5b7cf569e9e9791c019efabc7c6&oe=5FCAC2E0', key: '1'},
+        {name: 'Dawson Buist', category: 'Education', goal: 'I want to read one book every week', hobby: 'Hacking', email: 'coolguy@yeet.com', number: 1234573885, pic: 'https://scontent-ort2-2.xx.fbcdn.net/v/t1.0-9/104948192_4184601054913338_953146845638702540_n.jpg?_nc_cat=104&ccb=2&_nc_sid=09cbfe&_nc_ohc=7hwMelyT4SUAX9Bp9C0&_nc_ht=scontent-ort2-2.xx&oh=d51a7304d76517e63252d30b02fdb9e4&oe=5FC86C3A', key: '2'},
+        {name: 'Kelsey Yen', category: 'Spiritual', goal: 'I want to rest every day for an hour', hobby: 'Engineering', email: 'coolguy@yeet.com', number: 1234573885, pic: 'https://scontent-ort2-2.xx.fbcdn.net/v/t1.0-9/48361196_2323280584559505_806143671874355200_n.jpg?_nc_cat=109&ccb=2&_nc_sid=09cbfe&_nc_ohc=Vsd6KMQHRwoAX8Jwzit&_nc_ht=scontent-ort2-2.xx&oh=b0df354626d6db23c6e2d011ca9c7171&oe=5FC87E95', key: '3'},
+        {name: 'Belina Sainju', category: 'Relaxation', goal: 'I want watch TV every day', hobby: 'Reading', email: 'coolguy@yeet.com', number: 1234573885, pic: 'https://scontent-ort2-2.xx.fbcdn.net/v/t1.0-9/94386001_3052697268125747_8543896262028558336_o.jpg?_nc_cat=108&ccb=2&_nc_sid=09cbfe&_nc_ohc=HB2iMgmeyiMAX-xH8b0&_nc_ht=scontent-ort2-2.xx&oh=80085c34f488d26877a9d5e5cebc7074&oe=5FC74D3E', key: '4'},
```

Comment:
Ok, technically "relaxation" isn't a category...is it "leisure"? Should "leisure" be changed to "relaxation"?